src/ragger/navigator/navigation_scenario.py

```diff
@@ -31,7 +31,10 @@ def __init__(self, device: str, use_case: UseCase, approve: bool):
                 self.pattern = "^Confirm$"
 
             elif use_case == UseCase.TX_REVIEW:
-                self.navigation = NavInsID.USE_CASE_REVIEW_TAP
+                if device.startswith("stax"):
```

Comment:
```suggestion
                if device == "stax":
```
using `startswith` should probably only used on `nano`...

Reply:
Moreover I'm not sure in the first place why we are using strings when there is an enum.